src/coreclr/tools/Common/Compiler/CompilerTypeSystemContext.cs

```diff
@@ -184,9 +184,13 @@ public static unsafe PEReader OpenPEFile(string filePath, out MemoryMappedViewAc
 
         private EcmaModule AddModule(string filePath, string expectedSimpleName, bool useForBinding, ModuleData oldModuleData = null, bool throwOnFailureToLoad = true)
         {
-            filePath = Path.GetFullPath(filePath);
+            if (filePath != null)
```

Comment:
```suggestion
            if (filePath is not null)
```

Reply:
In the same method, string null check is done like below
```
if (expectedSimpleName != null && !simpleName.Equals(expectedSimpleName, StringComparison.OrdinalIgnoreCase))
```